### ISSUE TITLE
Mejoras en conceptos teóricos

### ISSUE THREAD

**Issue by Tulmot** [Tulmot/Sklearn-Multilabel#117]
Mejoramos los conceptos teóricos de Scikit-Learn, Multi-Label y Ensembles.

**Comment by Tulmot** [comment]
Tengo el término Ensemble y Multi-Label, he pensado que de multi-label podría meterlo dentro del concepto _Ensemble_ no?

**Comment by jjrodriguez** [comment]
Hay ensembles que no son para multi-label, y los problemas multi-label se pueden abordar con métodos que no son ensembles. Así que yo no veo que uno esté dentro del otro, aunque haya una intersección.

**Comment by alvarag** [comment]
Adjunto comentarios a memoria y anexos: sigue faltando mucha documentación que hacer...
[anexosC.pdf](https://github.com/Tulmot/Sklearn-Multilabel/files/1649938/anexosC.pdf)
[memoriaC.pdf](https://github.com/Tulmot/Sklearn-Multilabel/files/1649939/memoriaC.pdf)
